openmdao/docs/openmdao_book/other_useful_docs/multipoint_beam_opt.py

```diff
@@ -1,6 +1,8 @@
-
 import numpy as np
 
+om.display_source("openmdao.test_suite.test_examples.beam_optimization.beam_group.BeamGroup")
```

Comment:
What's the reason for printing these to stdout when this file is run (or imported from)? I think this is just used for the om command doc.

Reply:
fixed